src/status_im/ui2/screens/chat/components/message_home_item/view.cljs

```diff
@@ -129,7 +129,7 @@
          (get-in last-message [:content :text])]
         [render-subheader (get-in last-message [:content :parsed-text])])]
      (if (> unviewed-mentions-count 0)
-       [info-count unviewed-mentions-count {:top 16}]
+       [info-count/info-count unviewed-mentions-count {:top 16}]
```

Comment:
now we have quo core and it should be used instead i believe, like `quo/info-count`